ykcompile/src/lib.rs

```diff
@@ -1126,4 +1125,17 @@ mod tests {
         assert_eq!(inputs.0, 7);
         assert_eq!(inputs.0, args.0);
     }
+
+    #[test]
+    fn test_trace_inputs() {
+        let mut inputs = trace_inputs((1, 2, 3));
+        let th = start_tracing(Some(TracingKind::HardwareTracing));
+        inputs.0 = unsafe { add6(inputs.0, inputs.1, inputs.2, 4, 5, 6) };
+        let sir_trace = th.stop_tracing().unwrap();
+        let tir_trace = TirTrace::new(&*sir_trace).unwrap();
+        let mut args = (1,2,3);
+        TraceCompiler::<&(u64, u64, u64)>::compile(tir_trace).execute(&mut args);
+        assert_eq!(inputs.0, 21);
+        assert_eq!(inputs.0, args.0);
```

Comment:
Let's add another trace execution to the end of this test, with arguments different from those at trace time?